inductiva/simulation/simulator.py

```diff
@@ -39,29 +57,25 @@ def run(
             **kwargs: Additional keyword arguments to be passed to the
                 simulation API method.
         """
-        input_dir = files.resolve_path(input_dir)
-        if not input_dir.is_dir():
-            raise ValueError(
-                f"The provided path (\"{input_dir}\") is not a directory.")
-
-        if output_dir is None:
-            output_dir = input_dir.with_name(f"{input_dir.name}-output")
-            output_dir = files.get_timestamped_path(output_dir)
+        input_dir, output_dir = self._setup_dirs(input_dir, output_dir)
+        if pipeline is None:
+            params = [kwargs]
         else:
-            output_dir = files.resolve_path(output_dir)
+            params = [command.get_args() for command in pipeline]
```

Comment:
How is the command name passed? Shouldn't we pass `command.build()` for each command in the pipeline, where `build()` would concatenate the command name and its flags?

Reply:
This is done in the backend. I think here, we just want to let users define the method and the flags to run.